ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/viewmodel/CustomerCenterViewModel.kt

```diff
@@ -29,23 +40,77 @@ internal class CustomerCenterViewModelImpl(
         private const val STOP_FLOW_TIMEOUT = 5_000L
     }
 
-    override val state = flow {
+    private val _state = MutableStateFlow<CustomerCenterState>(CustomerCenterState.Loading)
+    override val state: StateFlow<CustomerCenterState> = flow {
```

Comment:
Hmm having both a `_state` and a `state` can be quite confusing... in general, this paradigm is to only make the private property not be mutable outside. So internally we use `_state` `MutableStateFlow` and we create a public `StateFlow` like: `_state.asStateFlow()`

Reply:
oh yes I had reverted this and forgot to push. But I am not seing the correct behavior and it's always staying as Loading. I was trying to debug with @JayShortway and coudln't figure it out 🤔